### ISSUE TITLE
Add profile picture

### ISSUE THREAD

**Issue by cacharle** [mszep/pandoc_resume#98]
It would be nice to have the possibility to add a profile picture on the left of the name and short description.

I managed to get an okay result with the following:

```md
<img src="../profile-picture.jpg" style="width: 170px; float: left; border-radius: 2px;"/>

My Name
=======

---
> Some description
---

<div style="clear: left;"></div>

...
```

![2023-11-25-164852_838x321_scrot](https://github.com/mszep/pandoc_resume/assets/25375061/6aae34a0-0a0f-4e1e-acea-32e9d14b0577)

**Comment by mszep** [comment]
Nice, but I'm guessing that wouldn't work for pdf output?

**Comment by cacharle** [comment]
Indeed, it doesn't. Was hoping you had more pandoc knowledge than me and you would have an idea 😅

**Comment by cacharle** [comment]
For anyone stumbling on this one, there is a stupidly simple work around for this, you just `Ctrl+p` in your browser. It will keep the pictures but also the colors

**Comment by tex** [comment]
For PDF you need to change template to this:

![Image](https://github.com/user-attachments/assets/95f90255-fe67-4e58-bb33-39ab1a4b0166)

Probably similar (use template and do what you did manually) could work for html.

In markdown, you need to specify this metadata:

![Image](https://github.com/user-attachments/assets/dcbc89fa-9adb-4f96-8479-05b3eeedb1dc)
